dashboard/src/main/resources/templates/dashboard.ftl

```diff
@@ -73,14 +73,21 @@
       Otherwise replace the dependency with something else.
     </p> 
     
+    <#assign x = 0>
     <ul id="unstable">
       <#list latestArtifacts as artifact, version>
+        <#assign x++>
```

Comment:
Why is this `x++` not inside the `if` condition?

Would it be easier to pass in a boolean to the template so there is less logic in it?

Reply:
This is  a local variable. Passing it in from java code couple the two more tightly together.